### PR TITLE
Murat_Test_1 Uploaded

### PR DESCRIPTION
**Pull request by Kkmyrat**
Ticket : murat_1
I completed my day 1, just changed code1 and code2
Thanks!
Don't mind picture pls :)
Screenshot 2022-09-14 183819

### CODE REVIEW

**Review by halilarslan1999 — CHANGES_REQUESTED** [review]
line 6 must be deleted

File: src/test/java/Murat_1/Murat_Test_1.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package Murat_1;
+
+public class Murat_Test_1 {
+
+    // code 1
+    // code 1
```
Comment (sumermehmet2001) [inline src/test/java/Murat_1/Murat_Test_1.java:6]: hi murat, is this code1 used twice? is it necessary? thanks

**Review by Kkmyrat — COMMENTED** [review]
Pls look at to ticket and you can see line 6 is should be there!
Thnx!

**Comment by halilarslan1999** [comment]
> 

after checking it again I saw that line 6 must be stay

File: src/test/java/Murat_1/Murat_Test_1.java (hunk 1 above)
Comment (sumermehmet2001) [inline src/test/java/Murat_1/Murat_Test_1.java:6]: hi this code is used twice

**Review by halilarslan1999 — APPROVED** [review]
confirmed

File: src/test/java/Murat_1/Murat_Test_1.java (hunk 1 above)
Comment (halilarslan1999) [reply src/test/java/Murat_1/Murat_Test_1.java:6]: I thik code 2 must be also reviewed

**Review by halilarslan1999 — APPROVED** [review]

File: src/test/java/Murat_1/Murat_Test_1.java (hunk 1 above)
Comment (Kkmyrat) [reply src/test/java/Murat_1/Murat_Test_1.java:6]: Hi, Mehmet bey
yea, this code is necessary! pls check ticket !

Comment (Kkmyrat) [reply src/test/java/Murat_1/Murat_Test_1.java:6]: because should be used!

Comment (sumermehmet2001) [reply src/test/java/Murat_1/Murat_Test_1.java:6]: all clear. agree with you.

Comment (sumermehmet2001) [reply src/test/java/Murat_1/Murat_Test_1.java:6]: if you used than it is ok

**Review by sumermehmet2001 — APPROVED** [review]

**Pull request merged**